models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,137 @@ class pp_cond_exp_mc_urbanczik_parameters
   double tau_syn_in[ NCOMP ]; //!< Rise time of inhibitory synaptic conductance in ms
 };
 
+/* BeginUserDocs: neuron, point process, conductance-based
 
-/** @BeginDocumentation
-@ingroup Neurons
-@ingroup pp
-@ingroup cond
-
+Short description
++++++++++++++++++
 
-Name: pp_cond_exp_mc_urbanczik - Two-compartment point process neuron with conductance-based synapses
+Two-compartment point process neuron with conductance-based synapses
 
-Description:
+Description
++++++++++++
 
 pp_cond_exp_mc_urbanczik is an implementation of a two-compartment spiking
 point process neuron with conductance-based synapses as it is used
-in [1]. It is capable of connecting to a Urbanczik synapse.
+in [1]_. It is capable of connecting to an :doc:`Urbanczik synapse
+<urbanczik_synapse>`.
 
-The model has two compartments: soma and dendrite, labeled as s and p, respectively.
-There is one excitatory and one inhibitory conductance-based synapse onto each compartment.
-Each compartment can also receive current input from a current generator, and an external
-(rheobase) current can be set for each compartment.
+The model has two compartments: soma and dendrite, labeled as s and p,
+respectively. There is one excitatory and one inhibitory conductance-based
+synapse onto each compartment. Each compartment can also receive current input
+from a current generator, and an external (rheobase) current can be set for
+each compartment.
 
 Synapses, including those for injection external currents, are addressed through
 the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
-
-Parameters:
+.. _multicompartment-models:
```

Comment:
A reference to multicompartment model documentation should also be linked from `iaf_cond_alpha_mc` neuron model. Or a general documentation needs to be created for multicompartment models which could be then linked from neuron models.

Reply:
Done.

Reply:
I would suggest to link this from the "Guides" page on RtD as well.